packages/block-editor/src/components/block-list/breadcrumb.native.js

```diff
@@ -16,20 +17,23 @@ import { View, Text, TouchableOpacity } from 'react-native';
  */
 import BlockTitle from '../block-title';
 import SubdirectorSVG from './subdirectory-icon';
-
 import styles from './breadcrumb.scss';
 
-const BlockBreadcrumb = ( { clientId, blockIcon, rootClientId, rootBlockIcon } ) => {
+const BlockBreadcrumb = ( { clientId, blockIcon, rootClientId, rootBlockIcon, isUnregisteredBlock } ) => {
 	return (
 		<View style={ styles.breadcrumbContainer }>
-			<TouchableOpacity style={ styles.button } onPress={ () => {/* Open BottomSheet with markup */} }>
+			<TouchableOpacity
+				style={ styles.button }
+				onPress={ () => {/* Open BottomSheet with markup */} }
+				disabled={ true } /* Disable temporarily since onPress function is empty */
+			>
 				{ rootClientId && rootBlockIcon && (
 					[ <Icon key="parent-icon" size={ 20 } icon={ rootBlockIcon.src } fill={ styles.icon.color } />,
 						<View key="subdirectory-icon" style={ styles.arrow }><SubdirectorSVG fill={ styles.arrow.color } /></View>,
 					]
 				) }
 				<Icon size={ 24 } icon={ blockIcon.src } fill={ styles.icon.color } />
-				<Text style={ styles.breadcrumbTitle }><BlockTitle clientId={ clientId } /></Text>
+				<Text style={ styles.breadcrumbTitle }>{ isUnregisteredBlock ? __( 'Unsupported' ) : <BlockTitle clientId={ clientId } /> }</Text>
```

Comment:
You don't need any custom handling here. Instead you can change the block title for the native `core/missing` to `Unsupported`

Reply:
So, can I change it in that [file](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/missing/index.js#L19) or should overwrite [webSettings](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/missing/index.native.js#L14) and get `Unsupported` only for native?

Reply:
So, can I change `title` it [that](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/missing/index.js#L19) file or overwrite `webSettings` only for native in that [file](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/missing/index.native.js#L14)?

Reply:
Yes, I think that should be fine